tests/system/Language/LanguageTest.php

```diff
@@ -356,4 +356,13 @@ public function testLanguageKeySeparatedByDot(): void
         $this->assertSame('The fieldname field is very short.', $this->lang->getLine('Foo.bar.min_length1', ['field' => 'fieldname']));
         $this->assertSame('The fieldname field is very short.', $this->lang->getLine('Foo.baz.min_length3.short', ['field' => 'fieldname']));
     }
+
+    public function testLanguageKeyWithDotAsNonMetacharacter(): void
+    {
+        $this->lang = Services::language('ab-CD', false);
+
+        $this->assertSame('File with dot.', $this->lang->getLine('Allin.file.with\..dot'));
+        $this->assertSame('The string with the dot at the end.', $this->lang->getLine('Allin.level_one.The string with the dot at the end\.'));
```

Comment:
I would like to write as `'getLine('Allin.level_one.The string with the dot at the end.')`.